testing/python_lib/integration_base.py

```diff
@@ -155,11 +155,23 @@ def parallelize(self, target, target_args):
             process = multiprocessing.Process(target=target, args=arg_tuple)
             jobs.append(process)
 
-        for job in jobs:
-            job.start()
+        batch_start = 0
+        if not batch_size:
```

Comment:
generally best to avoid modifying a varaible, and rather create a new one, e.g. 

batch_size_actual = batch_size or len(jobs)

the reason is it makes the code easier to understand, because I can understand the beginning, and the end, without having to understand the middle.

Reply:
Changed.